src/driver/oracle/OracleQueryRunner.ts

```diff
@@ -597,7 +598,7 @@ export class OracleQueryRunner extends BaseQueryRunner implements QueryRunner {
         const table = tableOrName instanceof Table ? tableOrName : await this.getCachedTable(tableOrName);
         const oldColumn = oldTableColumnOrName instanceof TableColumn ? oldTableColumnOrName : table.columns.find(c => c.name === oldTableColumnOrName);
         if (!oldColumn)
-            throw new TypeORMError(`Column "${oldTableColumnOrName}" was not found in the ${this.escapePath(table)} table.`);
+            throw new TypeORMError(`Column ${oldTableColumnOrName} was not found in the ${this.escapePath(table)} table.`);
```

Comment:
This is an error message, not a query.  Why change it?

Reply:
Fair point, simply caught up in the refactoring work. Will revert this part